### PR TITLE
Change defulat GPTQ config values

### PR DESCRIPTION
**Pull request by reuvenperetz**

### CODE REVIEW

File: model_compression_toolkit/gptq/common/gptq_config.py [hunk 1]
```diff
@@ -17,8 +17,8 @@
 from model_compression_toolkit.core.common.defaultdict import DefaultDict
 from model_compression_toolkit.core import common
 
-MAX_LSBS_CHANGE_MAP = {8: 4,
-                       4: 2,
+MAX_LSBS_CHANGE_MAP = {8: 1,
```
Comment (haihabi) [inline model_compression_toolkit/gptq/common/gptq_config.py:20]: change 8:1 to 8:2

File: model_compression_toolkit/gptq/common/gptq_config.py [hunk 2]
```diff
@@ -85,7 +85,7 @@ def __init__(self,
                  eps: float = 1e-6,
                  use_jac_based_weights: bool = False,
```
Comment (haihabi) [inline model_compression_toolkit/gptq/common/gptq_config.py:86]: use_jac_based_weights set to true

File: model_compression_toolkit/gptq/keras/quantization_facade.py [hunk 3]
```diff
@@ -84,13 +88,16 @@ def get_keras_gptq_config(n_iter: int,
             The configuration can be passed to :func:`~model_compression_toolkit.keras_post_training_quantization` in order to quantize a keras model using gptq.
 
         """
-
+        bias_optimizer = tf.keras.optimizers.Adam(learning_rate=LR_BIAS_DEFAULT)
```
Comment (haihabi) [inline model_compression_toolkit/gptq/keras/quantization_facade.py:91]: Change to SGD with a learning rate set to 1e-4

File: model_compression_toolkit/gptq/keras/quantization_facade.py [hunk 4]
```diff
@@ -47,12 +51,12 @@
...
-    def get_keras_gptq_config(n_iter: int,
+    def get_keras_gptq_config(n_iter: int = 5000,
                               optimizer: OptimizerV2 = tf.keras.optimizers.Adam(learning_rate=LR_DEFAULT),
-                              optimizer_rest: OptimizerV2 = tf.keras.optimizers.Adam(),
-                              loss: Callable = multiple_tensors_mse_loss,
+                              optimizer_rest: OptimizerV2 = tf.keras.optimizers.Adam(learning_rate=LR_REST_DEFAULT),
+                              loss: Callable = GPTQMultipleTensorsLoss(),
                               log_function: Callable = None,
-                              train_bias: bool = True) -> GradientPTQConfig:
+                              train_bias: bool = False) -> GradientPTQConfig:
```
Comment (haihabi) [inline model_compression_toolkit/gptq/keras/quantization_facade.py:59]: Remove train_bias flag

File: model_compression_toolkit/gptq/keras/quantization_facade.py [hunk 5]
```diff
@@ -32,13 +32,17 @@
 from model_compression_toolkit.core.common.target_platform.targetplatform2framework import TargetPlatformCapabilities
 
 LR_DEFAULT = 0.2
```
Comment (haihabi) [inline model_compression_toolkit/gptq/keras/quantization_facade.py:34]: set LR to 0.05

File: model_compression_toolkit/gptq/keras/gptq_loss.py [hunk 6]
```diff
@@ -66,3 +66,88 @@ def multiple_tensors_mse_loss(y_list: List[tf.Tensor],
...
+        loss_values_list.append(point_loss)
+        bias_loss_list.append(bias_loss)
+    if weights_for_average_loss is not None:
+        return tf.reduce_sum(weights_for_average_loss * tf.stack(loss_values_list)), \
+               tf.reduce_mean(tf.stack(bias_loss_list))
+    else:
+        return tf.reduce_mean(tf.stack(loss_values_list)), tf.reduce_mean(tf.stack(bias_loss_list))
+
+class GPTQMultipleTensorsLoss:
+    def __init__(self, norm_loss: bool = False):
```
Comment (haihabi) [inline model_compression_toolkit/gptq/keras/gptq_loss.py:119]: Change to true

File: model_compression_toolkit/gptq/keras/quantization_facade.py [hunk 7]
```diff
@@ -47,12 +51,12 @@
     DEFAULT_KERAS_TPC = get_target_platform_capabilities(TENSORFLOW, DEFAULT_TP_MODEL)
 
 
-    def get_keras_gptq_config(n_iter: int,
+    def get_keras_gptq_config(n_iter: int = 5000,
```
Comment (haihabi) [inline model_compression_toolkit/gptq/keras/quantization_facade.py:54]: Remove default

File: tests/keras_tests/function_tests/test_get_gptq_config.py [hunk 8]
```diff
@@ -59,7 +59,7 @@ def test_get_keras_gptq_config(self):
                                 weights_bias_correction=False)  # disable bias correction when working with GPTQ
         gc = mct.GumbelConfig(temperature_learning=True)
         gptq_configurations = [get_keras_gptq_config(n_iter=1),
-                               get_keras_gptq_config(n_iter=1, train_bias=False),
```
Comment (haihabi) [inline tests/keras_tests/function_tests/test_get_gptq_config.py:62]: Why you remove the train_bias=False ?

Comment (reuvenperetz) [reply tests/keras_tests/function_tests/test_get_gptq_config.py:62]: We decided to remove it from the API

Comment (haihabi) [reply tests/keras_tests/function_tests/test_get_gptq_config.py:62]: Right

**Review by haihabi — APPROVED** [review]
